### DISCUSSION TITLE
Story/campaign concept and a loose, proposed mission framework centered around a rogue planet discovered in contested space

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10173] [Show and tell]
Morning all, 

Got into the game early this year and I’ve really enjoyed the story and lore side of it. Having had a look on the discussions board, it sounds like there’s still demand for mission/story writing, so I thought I’d give it a go. 

Currently also working on getting the hang of writing (coding?) playable missions, so with any luck I should be able to contribute more on that front as well soon. 

Would really appreciate any thoughts, opinions and feedback etc

Ahead, MAIN GAME SPOILERS ABOUND

This draft assumes that the player has made contact with the Kor Efreti and made significant headway in the Wanderer storyline, but it would be pretty easy to adapt it to be playable once the player has only made contact with the Remnant if it might be of more interest earlier in a typical play-through.

Campaign: Welcome to Parivrajaka

Story/plot:
A rogue planet has appeared on the fringes of human space - an area on the border of Free Worlds, Republic and Syndicate territory. Despite its close proximity it is still only accessible by jump drive.

The player arrives, only to be met by representatives of the Syndicate and the Republic, confirming that both have been amassing a supply of jump drives in secret, while the player is interpreted as representing the Free Worlds by the other two factions. All three have detected energy signatures from the surface that suggest advanced technology is present. After a brief standoff, all three land to explore the planet.

The planet supports an atmosphere and, even more shockingly, is home to strange plant-life clustered around volcanic fissures across the surface. While the player is exploring, they hear an explosion in the distance followed by the sound of ships taking off. They are forced to flee back to their own ship where they must fight off a fleet of unidentified vessels from the planet - any they disable and attempt to board self-destruct. 

The captain rescues the Syndicate and Republic ships and returns to Bourne to report back to the free-worlds. 

The Council asks for an escort to Sol to attend a summit between the Free Worlds, Syndicate and Republic. After a brief but intense skirmish with an Oathbreaker fleet (led by one Commander Park, who holds a vendetta against the playerin particular) the Council arrives on Earth where negotiations quickly break down. The Republic presumes sovereignty over the new planet, the Syndicate tries to claim salvage and resource rights and the Free Worlds delegation refuses trust either of them - due to perceived inaction against the Oathbreakers by the Republic, and due to rumours of extremist elements still at work within the Syndicate. 

While the negotiations are fruitless, one of the council diplomats manages to obtain scans taken by an undercover Deep Security operative on board the Syndicate ship of wreckage recovered from the initial battle against the ships from Parivrajaka, confirming that it is not technology the player is familiar with. 

The conference is suddenly abandoned when reports of activity come in from Parivrajaka. The player arrives at the planet accompanied by a handful of Syndicate and Republic ships to find a fleet of the alien ships are now present throughout the ‘system’. Those beyond the atmosphere aren’t hostile but when the human ships attempt to land on the surface they are fired on and forced to retreat. 

The human fleet is able to detect a transmission coming from the surface but cannot translate the alien language, leaving the captain to visit various alien species they have made contact with to see if they can help translate. 

The Kor Efreti seem to be familiar with the scans, but lost much of their ancient historical texts in the chaos of the civil war and can’t provide a translation. The player is forced to raid a heavily armed Korath Exile ship in remnant space to steal a copy of their database.

The player returns to Efreti space with the database and discovers that there are fragmented records of Korath raids on Parivrajaka over 100,000 years ago as it drifted close to Korath space, with several of their ships being captured or disappearing before contact with the planet was lost. 

Returning to human space, the captain has to fend off a party of vengeful Korath, outraged at the theft of their database. After emerging victorious, the captain returns to Parivrajaka and attempts to make contact with the ships in orbit via a pre-written message provided by the Kor Efreti. 

The Captain tries to contact the alien ships using the pre-recorded Korath “hello”, but this causes the ships to turn hostile. More significantly, a planet-based weapon platform activates and begins firing at all the human ships in orbit. Still unable to land and with the planet-based cannon an overwhelming threat, the Syndicate and Republic ships withdraw to avoid losing their precious jump drives. Even the captain has little choice but to do the same - the aliens can’t penetrate their cloaking device, but they have no way to land in order to show the aliens that they mean them no harm and (hopefully) persuade them to stop shooting.

When they next speak to the Remnant, the player updates them on the latest news and is only a little surprised when they reveal they have been monitoring the planet from cloaked ships since it first appeared. This has given them an opportunity to study the planet's defense systems and identify an opportunity to slip through to the surface.

Accompanied by a small fleet of Remnant ships, the player returns to Parivrajaka and lands on the surface. While the Remnant decide to stay concealed, the Captain finally meets the inhabitants of the planet - the Kytin. 

After a tense few moments, the aliens seem to accept that the Captain means them no harm - the player surmises that the Kytin understood their initial message but didn’t trust them, perhaps believing that the humans were allied with the Korath. 

Before there’s time for any of the usual niceties of first-contact, the player’s ship detects hostile vessels in orbit - Commander Park has arrived with a fleet of Navy vessels alongside Korath scouts and raiders captured from the pirate systems. If the Kytin are already suspicious about the relationship between humans and the Korath, this could make matters worse…

Fortunately, siding with the Kytin during the battle is enough to convince them that you, at least, aren’t a threat to them, and with the combined forces of the Kytin, the player and the Remnant ships that decloak and come to your assistance, you defeat Commander Park and his fleet. 

The Kytin, unfortunately, are still wary of humans and are unwilling to grant you a license to purchase any of their technology, but you wouldn’t put it past the Remnant to have found time during the battle to lay their hands on some Kytin tech floating through the debris field. Of course, the Kytin may just as easily have found a few jump drives among the wreckage of the Oathbreaker fleet…

Epilogue:
The Kor Efreti, grateful to have access to parts of their history they feared was lost forever, offer to negotiate on your behalf with the Korath Exiles to prevent further revenge attacks. 
The captain is able to make contact with the Korath in a neutral system where the Efreti explain that you were not attempting to steal Korath secrets and handed the database over to them. 
The Exiles remain hostile (as always) but no longer actively hound you throughout human space. 

Expanded campaign:
I figured this could work as a standalone mission-chain or a longer campaign centered around Parivrajaka and the Kytin with rewards in the form of access to purchase new outfits and maybe acquire one or two unique H2H, staggered through 3 main mission chains (the story above constituting the first).

Second mission chain would focus on a splinter group (the Terran Defence Force) forming from fringe elements within the Free Worlds, Republic and Syndicate focused on acquiring alien technology through any means necessary, claiming the need to protect humanity from alien threats in the wake of the Pug invasion and now the Parivrajaka crisis. The Captain would be attempting to recover elements of stolen Kytin technology with the option to return it to the Kytin themselves or the Remnant for different outcomes. The Terran Defence Forces would play a similar in-game role to the Oathbreakers (relatively high-tier 'pirate' fleets with access to some alien outfits and ships), focused in areas where the player is likely to encounter them during the mission chain. 

Third mission chain would be the conclusion, culminating in the Kytin forcing open a hyperspace lane to a neighbouring system (or a hyperspace lane to their system being forced open from outside). Culminates in a happy ending where the “Big Three” human factions agree to have diplomatic relations with the Kytin conducted by The Deep, who help the Kytin develop their own ion engines in exchange for giving the Deep limited access to their technology. 

Factions:
Kytin: Tier 1.5-1.8 (on-par with Remnant)
The Kytin are a semi-sedentary species sharing traits with certain plants and fungi. Able to connect to a planet-wide mycelial network, Native to Parivrajaka, their planet has drifted through the Milky way for millions of years, having never been caught in the orbit of a star.

The perpetual darkness of their sunless sky is punctuated with pulses and blooms of light among the steaming, jet black fungal rainforests, whether from the many natural volcanic fissures that release life-giving heat and energy into the atmosphere, or the bizarre bioluminescent wildlife. 
Large, warm shallow seas cover a third of the planet’s surface, glittering in the starlight on the rare occasions the heavy, humid atmosphere sees a brief break in the cloud layer. 

Their technology is biological in origin, making use of genetically engineering fungi in their ships and weapons. 

Strengths:
Organic-based hull repair that speeds up at the cost of additional power drain when the ship is closed to being disabled. 
Long range homing weapons that use self-replicating organic ammunition (similar concept to Pug weapons)
Chitinous-organic hull has built-in energy storage

Weaknesses:
Poor power generation - masters of geothermal power extraction, their spaceship reactors are bulky and inefficient by comparison.
Lack disruption technology - have trouble dealing with heavily shielded targets
Comparatively small engine space leave their heavy ships lacking in speed and maneuverability.

Outfits:
Weapons:
Damzavisa Fang
Damzavisa Tusk

Turrets:
Flak Basidium (Anti-missile)

Systems:
Regenerative Lamellae - Hull repair (large)
Regenerative Lamellae - Hull repair (small)

Power:
Calvatia biogenerator
Prototaxites biogenerator
Armillaria biogenerator

**Comment by DoublePants** [comment]
I think one thing you may need to consider for this is that the plan is to have a Republic and Syndicate mission chain at some point, so you'll need to come up with a few alternate narrative steps depending on which faction the player has joined that can be added when those mission chains are added.

**Reply by Vemenous-Repentile** [reply]
Thanks - I think I could adapt the narrative side of things to work depending on the different allegiances made by the player. Would probably make it more complicated to code, but worth doing.

**Reply by TheGiraffe3** [reply]
Wouldn’t be that hard using branches. It would take quite a bit of time though.

**Reply by Vemenous-Repentile** [reply]
> Wouldn’t be that hard using branches. It would take quite a bit of time though.

I figured it might be a bit time consuming to write all the paths. I'm pretty comfy with writing the 'copy' side of stuff, it's more the coding that I'd need to make sure I've got the hang of. Appreciate it would most likely take a lot longer than I'm estimating though

**Comment by TheGiraffe3** [comment]
The Republic and the Syndicate having Jump Drives doesn’t quite fit in with lore. You never see a Pug ship being boarded unless you board it. It’s possible that in a system like Caph that happens, in a place where first last doesn’t see it, but it’s rather unlikely.

**Reply by Vemenous-Repentile** [reply]
Don't the syndicate supply the player with a jump drive during the FW campaign (assuming they don't already have one) and  doesn't Admiral Danforth use one (not provided by the player) to join the player during the mission to Host?

I might be misremembering but I don't think that the player has to provide either faction with jump drives, but both show that they have access to them, albeit in implied limited supply? I thought it was implied that both factions have either captured them directly from the Korath or via pirates who captured them from Korath raider themselves, with additional examples acquired during the Pug invasion?

**Reply by DoublePants** [reply]
It also doesn't really make sense that no other human has ever gotten a jump drive, given how relatively easy it is for the player.

**Reply by TheGiraffe3** [reply]
You’re right about that. 
I suppose what I mean is the Republic and Syndicate are not going to be using their jump drives very often. If that Cruiser with a jump drive gets blown up, they just lost over a tenth of their Jump Drive stock. 
They basically are hoarding them.

**Reply by Vemenous-Repentile** [reply]
> You’re right about that. I suppose what I mean is the Republic and Syndicate are not going to be using their jump drives very often. If that Cruiser with a jump drive gets blown up, they just lost over a tenth of their Jump Drive stock. They basically are hoarding them.

Agreed - I got the impression that the question "who has access to jump drives" had a fairly flexible answer depending on the needs of the narrative - the Republic, the Syndicate, the Unfettered, the Bunrodea and the Remnant all apparently have a small but limited stockpile - enough that it's not totally inexplicable that the player might encounter them in areas inacessible via normal hyperdrive, but not so many that  it begs the question why we don't see whole fleets equipped with jump drives from those factions. 

In terms of the Syndicate and Republic being willing to use their jump drives to reach the rogue planet, I thought it was plausible that they'd be willing to do so given the potentially greater risk of letting someone else get their hands on whatever goodies that planet holds first.

**Reply by TheGiraffe3** [reply]
> Goodies that planet holds

Not sure the Republic would think that way. The Syndicate? Oh, definitely. 
I would think that the Republic would send the player in first to see if there’s anything worth it there, but the player is being shadowed by the Syndicate. If there’s advanced technology, there is probably weapons. If there’s weapons… well, as you said, the Syndicate does it, the Republic is cautious.

**Reply by Vemenous-Repentile** [reply]
> > Goodies that planet holds
> 
> Not sure the Republic would think that way. The Syndicate? Oh, definitely. I would think that the Republic would send the player in first to see if there’s anything worth it there, but the player is being shadowed by the Syndicate. If there’s advanced technology, there is probably weapons. If there’s weapons… well, as you said, the Syndicate does it, the Republic is cautious.

I got the sense that the 'Republic' (specifically the government itself) were a bit closer to the Syndicate in terms of their outlook - not quite so brazen when it comes to acting out of self-interest, but still erring on the cynical side. The way the Navy are characterised, on the other hand, I could absolutely see them erring on the side of caution and not looking to gain an 'unfair' advantage via underhand tactics like spying, sabotage, or sneaking in to steal alien tech

**Reply by TheGiraffe3** [reply]
And by Republic you mean Parliament? Because technically the Deep is included in the Republic.

**Reply by Vemenous-Repentile** [reply]
Yes -  should've said "Parliament"

**Comment by mOctave** [comment]
I quite like this story. It generally seems to be quite logical, but I have a few questions and comments.

1. Why has no one noticed the Kytin before? It seems to me that their presence should be fairly obvious if they are a technologically advanced spacefaring species, even if they're on a rogue planet. Are they intentionally stealthy? Does the mycellial network allow them to eschew radio, etc.?
2. Presumably they need connection to the mycellial network to live, right? How does this work with them having ships? Can they survive separated from the planet for a while? Or are all their ships drones?
3. Actually, why do they even need ships if they live on a rogue planet? They don't have asteroids to mine, presumably. Does their planet have a resource-rich moon? Did they build ships in response to the Korath invasion?
4. Regarding the planet-based cannon: The engine doesn't have support for planet-based weapons, so you'd probably have to use ships. You could probably do it with a bit of hackyness (make a new ship, give it a smoke cloud as a sprite, give it a weapon that kills it instantly when fired and has enough range to cross the system, add that ship to a fleet that gets spawned regularly between two events), but it would be hard to get into vanilla.
5. What would the response of the Drak, Quarg, and Pug be to this rogue planet? Does one of the groups already know about it? I would expect that at least the Drak and Pug would, probably the Quarg as well. Have either the Quarg or Pug tried to "claim" the Kytin? Is there a reason the Archons aren't keeping a close eye on the planet and warning off the humans who try to colonize it?

**Reply by Vemenous-Repentile** [reply]
Thanks for the questions!

1 - A bit sci-fi hand-wavy, but I tjhought perhaps it's only recently being detected as it has transited between a nearby star and a high powered telescope. As a lone planet without a parent star, I figured it would be difficult to detect using more conventional methods. As for the Kytin themselves I figured they would be a stealthy species based on their only past contact with outsiders - the Korath. As for their communications technologies, I didn't imagine them specifically eschewing more conventiontal radio etc, but it might be a cool idea to include now that you mention it - an extra barrier to communicating with the Kytin that the player has to overcome.
- Edit - Might be a fun mission to use a "Billboard" bulk carrier to deliver the initial message.

2 - I pictured them being "semi sedentary" e.g. they can disconnect from the mycelial network for a day or so before needing to reconnect to it to "feed". They don't have the means to "feed" aboard their ships (at least initially) as they've never had the ability to venture beyond their home 'system'. 
edit: I was keen to include plausible reasons why they wouldn't become ubiquitous in human space at the end of the campaign so as to avoid this storyline interfering with existing or upcoming game content as much as possible.

3 - I like the idea of a resource rich moon giving them motivation to build spacefaring ships, with their culture having become more militarised as a result of the Korath attacks. I had it in the back of my mind that the Kytin might not have developed hyperdrives given that their world has never been connected to a hyperspace lane. 

4 - I had somethign like the Drak ship that protects the void sprites in mind, but firing from a fixed position on the planet. If this is a logistical nightmare to implement then placing the weapon on board a large ship would probably work just as well. 

5 - I was wondering if/how to include those higher tier species in the plot, particularly the Quarg since the story touches on the topic of opening up a new hyperspace link to Parivrajaka. As to how they might get involved to help or hinder the player, I'm open to ideas. I could see them offering to protect the Kytin from outside interference - maybe they allow a new hyperspace link to Parivrajaka to stay open but agree to protect the Kytin from any unwelcome outside interference? As for the Drak I could definitely see them taking an interest (presumably acting via the Quarg). 

The Pug that invaded human space seemed (imho) to be trying to encourage humans to explore and to be prepared to defend themselves against alien threats, so perhaps they'd be quite pleased to watch humans interacting with a new, potentially hostile, species?
